### DISCUSSION TITLE
pac solution import option for dependency checking only

### DISCUSSION THREAD

**Discussion by alex-mcla** [microsoft/powerplatform-build-tools#492] [Ideas]
It would be useful on pac solution import to have an option for dependency checking only.

This would allow dependencies to be checked and result reported without importing the solution, which might not be wanted in all situations.

**Comment by rajyraman** [comment]
I like this! One more thing I would like to add is a "What-If" style of deployment or like how azd does it (https://devblogs.microsoft.com/azure-sdk/azure-developer-cli-azd-august-2023-release/#preview-provisioning-changes-with-azd-provision-preview) would be quite useful.

cc @kkanakas
